src/System.Net.Sockets.Legacy/tests/PerformanceTests/project.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.0-beta-*",
+    "System.Diagnostics.Debug": "4.0.10-beta-*",
+    "System.Runtime": "4.0.20-beta-*",
+    "System.Threading": "4.0.0-beta-*",
+    "System.Threading.Tasks": "4.0.0-beta-*",
+    "System.Threading.Tasks.Parallel":  "4.0.0-beta-*",
```

Comment:
Yes, anything where we have a stable package and all you need surface-area-wise is available in that stable package you should use the stable package.